Libplanet/Address.cs

```diff
@@ -135,24 +133,20 @@ public Address(string hex)
         }
 
         /// <summary>
-        /// Creates an <see cref="Address"/> instance from the given Bencodex <see cref="Binary"/>
-        /// (i.e., <paramref name="address"/>).
+        /// Creates an <see cref="Address"/> instance from given <paramref name="bencoded"/>.
         /// </summary>
-        /// <param name="address">A Bencodex <see cref="Binary"/> of 20 <see cref="byte"/>s which
+        /// <param name="bencoded">A Bencodex <see cref="Binary"/> of 20 <see cref="byte"/>s which
         /// represents an <see cref="Address"/>.
         /// </param>
-        /// <exception cref="ArgumentException">Thrown when the given <paramref name="address"/>
-        /// did not lengthen 20 bytes.</exception>
-        public Address(Binary address)
-            : this(address.ByteArray)
+        /// <exception cref="ArgumentException">Thrown when given <paramref name="bencoded"/>
+        /// is not an encoding of a <see cref="byte"/> array of length 20.</exception>
```

Comment:
```suggestion
        /// is not an encoded Bencodex <see cref="Binary"/> value of length 20.</exception>
```

Might be more clear.